examples/existing-kms/main.tf

```diff
@@ -0,0 +1,20 @@
+##############################################################################
+# Key Protect root key
+##############################################################################
+
+module "key_protect_root_key" {
+  source                  = "../.."
+  key_protect_instance_id = var.existing_kms_instance_guid
+  key_name                = "${var.prefix}-root-key"
+}
+
+##############################################################################
+# Key Protect standard key
+##############################################################################
+
+module "key_protect_standard_key" {
```

Comment:
Better to have module name as `kms` instead of `key_protect` in new example, as we are going to update this in main module too.